sdk/cosmos/azure-cosmos/test/test_full_text_policy.py

```diff
@@ -0,0 +1,264 @@
+                   in e.http_error_message
+
+        # Pass a full text policy with an unsupported path language
+        full_text_policy_wrong_default = {
+            "defaultLanguage": "en-US",
+            "fullTextPaths": [
+                {
+                    "path": "/abstract",
+                    "language": "spa-SPA"
+                }
+            ]
+        }
+        try:
+            self.test_db.create_container(
+                id='full_text_container',
+                partition_key=PartitionKey(path="/id"),
+                full_text_policy=full_text_policy_wrong_default
+            )
+            pytest.fail("Container creation should have failed for lack of embedding policy.")
+        except exceptions.CosmosHttpResponseError as e:
+            assert e.status_code == 400
+            assert "The Full Text Policy contains an unsupported language spa-SPA. Supported languages are:"\
+                   in e.http_error_message
+
+    def test_fail_create_full_text_indexing_policy(self):
+        full_text_policy = {
+            "defaultLanguage": "en-US",
+            "fullTextPaths": [
+                {
+                    "path": "/abstract",
+                    "language": "en-US"
+                }
+            ]
+        }
+        # Pass a full text indexing policy with a path not present in the full text policy
+        indexing_policy_wrong_path = {
+            "fullTextIndexes": [
+                {"path": "/path"}
+            ]
+        }
+        try:
+            self.test_db.create_container(
+                id='full_text_container',
+                partition_key=PartitionKey(path="/id"),
+                indexing_policy=indexing_policy_wrong_path,
+                full_text_policy=full_text_policy
+            )
+            pytest.fail("Container creation should have failed for lack of embedding policy.")
+        except exceptions.CosmosHttpResponseError as e:
+            assert e.status_code == 400
+            assert "The path of the Full Text Index /path does not match the path specified in the Full Text Policy"\
+                   in e.http_error_message
+
+        # Pass a full text indexing policy with a wrongly formatted path
+        indexing_policy_wrong_path = {
+            "fullTextIndexes": [
+                {"path": "abstract"}
+            ]
+        }
+        try:
+            self.test_db.create_container(
+                id='full_text_container',
+                partition_key=PartitionKey(path="/id"),
+                indexing_policy=indexing_policy_wrong_path
+            )
+            pytest.fail("Container creation should have failed for lack of embedding policy.")
+        except exceptions.CosmosHttpResponseError as e:
+            assert e.status_code == 400
+            assert "Full-text index specification at index (0) contains invalid path" in e.http_error_message
```

Comment:
Nit: Why we have completely different error message format with similar issue from `full text policy` ? Can we make them more uniformed? 

"The Full Text Policy contains an invalid Path: abstract"

Reply:
all of these errors come from the service in the backend - we don't fabricate any of them in the SDK. these checks are here to validate that they stay the same, or that if they change we will become aware of it